lua/rsync/project.lua

```diff
@@ -64,6 +65,12 @@ function project.get_config_table()
     -- decode config file and save to global table
     table = get_config(config_file_path)
     if table ~= nil then
+        -- fix the path if it is missing a /
+        local last_char = string.sub(table.remote_path, -1)
+        if last_char ~= nil and last_char ~= "/" then
+            table.remote_path = table.remote_path .. "/"
+        end
+
```

Comment:
Shouldn't this always be set to a path? When i had a path where the / at the end was missing i couldn't sync a file down because the path was concatenated incorrectly. 

Can you explain me why the user doesn't always wants this?

Shouldn't it always be set to a folder on the remote?